dev-python/pathtools/pathtools-0.1.2-r3.ebuild

```diff
@@ -0,0 +1,18 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} pypy)
+
+inherit distutils-r1
+
+DESCRIPTION="Pattern matching and various utilities for file systems paths"
+HOMEPAGE="https://pypi.org/project/pathtools/"
+SRC_URI="mirror://pypi/p/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~x86"
+
+distutils_enable_tests nose
```

Comment:
This does not run any tests as the source has none.

Reply:
Fixed